### DISCUSSION TITLE
Card_mod not working

### DISCUSSION THREAD

**Discussion by allseven** [piitaya/lovelace-mushroom#1365] [Q&A]
I have a card that's doing everything I want it to do except change the colour of the text. Can anyone say where I'm going wrong and/or what is missing please?

type: custom:mushroom-chips-card
chips:
  - type: template
    content: '{{ states(''sensor.wiser_lts_temperature_dining_room'') }} °C Dining room'
    icon: >-
      {% if is_state('climate.wiser_dining_room', 'hvac_action') == 'heating' %}
      mdi:fire  {% else %} mdi:fire-off  {% endif %}
    icon_color: >-
      {% if is_state('climate.wiser_dining_room', 'hvac_action') == 'heating' %}
      red  {% else %} green  {% endif %}
    card_mod:
      style: |
        ha-card {
          color: red;
        }

**Comment by BBE-FR** [comment]
Use this reference post:
https://community.home-assistant.io/t/mushroom-cards-card-mod-styling-config-guide/600472

![image](https://github.com/user-attachments/assets/9f8080dc-5572-4e6f-bbe8-bf8b12f3d3f8)
